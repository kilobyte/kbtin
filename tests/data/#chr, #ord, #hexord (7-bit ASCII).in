#chr x {98 108 97 104};#showme {$x}
#chr x U+0062 U+006C U+0061 U+0068;#showme {$x}
#chr x 98 0x6c 97 U+0068;#showme {$x}
#ord x blah;#showme {$x}
#hexord x blah;#showme {$x}
