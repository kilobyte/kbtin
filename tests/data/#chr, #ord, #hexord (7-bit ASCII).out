blah
blah
blah
98 108 97 104
U+0062 U+006C U+0061 U+0068
